TDTChocolate.podspec

```diff
@@ -3,16 +3,16 @@ Pod::Spec.new do |s|
   s.version  = '2.10.0'
   s.summary  = 'Library containing additions to Cocoa and Cocoa Touch'
   s.homepage = 'https://github.com/talk-to/Chocolate'
-  s.author   = 'Talk.to'
+  s.author   = 'Directi'
   s.license  = 'BSD'
   s.source   = {
     :git => 'https://github.com/talk-to/Chocolate.git',
     :tag => "#{s.version}"
   }
   s.requires_arc = true
 
-  s.ios.deployment_target = '7.0'
-  s.osx.deployment_target = '10.8'
+  s.ios.deployment_target = '8.0'
+  s.osx.deployment_target = '10.9'
```

Comment:
You could consider reverting that commit, because fixing a two deprecation warnings does not seem sufficient reason for dropping 7 support (unless you have something else in mind too which'll require dropping 7 support anyways). 

Moreso, since Ringo supports 7 and will be unable to use the updated Chocolate if you make this change.